docs/.vuepress/sidebar/en.ts

```diff
@@ -249,7 +249,7 @@ export const enSidebar = sidebar({
     {
       text: "Block Explorer", // required
       link: "/tools/block-explorer", // optional, which should be a absolute path.
-      children: ["/tools/block-explorer/block-explorer-menu", "/tools/block-explorer/contract-verification"],
+      children: ["/tools/block-explorer/block-explorer-menu", "/tools/block-explorer/contract-verification", "/tools/block-explorer/block-explorer-api"],
```

Comment:
If block-explorer-api is first on the page it should be first link?

Reply:
The API is the last subtitle on the block Explorer [readme file](https://aqwzx-zksync-v2-docs--pr772-add-block-explorer-a-zpgo5wmh.web.app/docs/tools/block-explorer/#block-explorer-api-1), even though the preview URL shows the [deleted part](https://aqwzx-zksync-v2-docs--pr772-add-block-explorer-a-zpgo5wmh.web.app/docs/tools/block-explorer/#block-explorer-api).

@Romsters mentioned having the API listed as the last one on the readme of Block Explorer and having the item as the last one. We can move it up.

Reply:
API should be the last one